### PR TITLE
VOW Tokens

### PR DESCRIPTION
**Pull request by mainman879**

### CODE REVIEW

File: tokens.xml [hunk 1]
```diff
@@ -1899,6 +1917,20 @@ When this creature dies, create a 2/2 red Dragon creature token with flying and
             <reverse-related>Nesting Dragon</reverse-related>
             <token>1</token>
             <tablerow>2</tablerow>
+        </card>
```
Comment (ebbit1q) [inline tokens.xml:1920]:
```suggestion
        </card>
        <card>
```

**Comment by mainman879** [comment]
Question for ordering cards when spaces are involved: Do we order it letter by letter (thereby ignoring the spaces), or word by word? The reason I ask is because of:

- <reverse-related>Bloodcrazed Socialite</reverse-related>
- <reverse-related>Bloodsoaked Reveler</reverse-related>
- <reverse-related>Bloodtithe Harvester</reverse-related>
- <reverse-related>Bloodvial Purveyor</reverse-related>
- <reverse-related>Blood Petal Celebrant</reverse-related>

The Blood Petal Celebrant has a space but the others dont, so if we go letter by letter it goes between bloodcrazed and bloodsoaked, and if we go word by word it goes before all of them.

**Comment by ebbit1q** [comment]
space is a character, it is a lower value than others

**Comment by tooomm** [comment]
To clarify on your statement, is A a lower value than B? Or the other way around?

**Comment by mainman879** [comment]
Known issue: There is no token image for Timothar, Baron of Bats. It is a 1/1 Bat token with flying and another effect. I could not find an image of this token with the other tokens.

**Comment by ebbit1q** [comment]
if no placeholder art can be found for the card you should remove the picURL alltogether

**Comment by ebbit1q** [comment]
actually, timothar specifies that the card gains the ability, this means it is not part of the card's text box imo

**Pull request merged**